code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -10,21 +10,20 @@
 	explosion(target, devastation_range = -1, light_impact_range = 2, explosion_cause = src)
 	return BULLET_ACT_HIT
 
-/// PM9 HEDP rocket
+/// PM9 standard HE rocket
 /obj/projectile/bullet/a84mm
```

Comment:
can you rename this to rocket? annoying to var edit

Reply:
Just the path? Sure thing.